frontend/components/siteList/siteListItem.js

```diff
@@ -34,11 +34,8 @@ const SiteListItem = ({ site }) =>
       <h3>
         <Link to={`/sites/${site.id}`} title="View site settings">
           { site.owner }/{ site.repository }
-        </Link>
-
-        <GitHubRepoLink owner={site.owner} repository={site.repository} />
+        </Link>&nbsp;<GitHubRepoLink owner={site.owner} repository={site.repository} />
```

Comment:
another "trick" you can use in React instead of using `&nbsp;` is `{' '}`:

```jsx
...
</Link>
{' '}
<GitHubRepoLink owner={site.owner} repository={site.repository} />
```

Reply:
fixed 👍